src/main/java/seedu/address/model/shift/RoleRequirement.java

```diff
@@ -49,16 +75,28 @@ public static boolean isValidRoleRequirement(String test) {
         return test.matches(VALIDATION_REGEX);
     }
 
+    /**
+     * Returns true if the quantity filled has not reached the quantity required.
+     */
+    public boolean isFilled() {
+        assert quantityFilled <= quantityRequired;
```

Comment:
Would it be more meaningful to put this assertion in the constructor instead? Just a suggestion, not sure if it's good practice

Reply:
Makes sense, edited